gradle.properties

```diff
@@ -1,2 +1,2 @@
 group=com.heapwhisperer.metrics.jvm
-version=1.0.2
+version=1.0.2-rocana2.0.0-SNAPSHOT
```

Comment:
What about just `rocana1`, then if we issue another build we make it `rocana2`?
Or if that's not granular enough: `rocana-1.0.0`, meaning, the 1.0.0 release of this lib?

p.s. I'm not going to let this hold up the PR, I just wanted to start the discussion